### DISCUSSION TITLE
Is there any option to whitelist or blacklist character in PaddleOcr

### DISCUSSION THREAD

**Discussion by arhamshah** [PaddlePaddle/PaddleOCR#7515] [General]
In pytesseract, there is an option to whitelist or blacklist the characters. I get '工' instead of '1' so I wanted to know whether it supports or are there any other options available to overcome this issue ?

**Comment by Evezerest** [comment]
for the English scenario, please use the English OCR model `ocr = PaddleOCR(use_angle_cls=True, lang="en") `

**Reply by arhamshah** [reply]
Okk thanks @Evezerest, can you help me on this https://github.com/PaddlePaddle/PaddleOCR/issues/7594 issue too?
